_1327/static/less/1327.less

```diff
@@ -354,6 +354,10 @@ nav li:hover > .dropdown-menu {
 .container.content {
 	padding-left: 20px;
 	margin-bottom: 30px;
+	// Wraps all overflowing text in the content section, regardless whether it's part
+	// of the main text, of the attachments list or table or any other content element
+	word-wrap: break-word;
+	word-break: break-all;
```

Comment:
this makes the line break always exactly at the end of the container even if a word could entirely be placed onto the next line and would make the break unnecessary (see https://developer.mozilla.org/en-US/docs/Web/CSS/word-break).